js/controller.js

```diff
@@ -1598,28 +1602,32 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       }
```

Comment:
we need an extra check in here to see if 'element' is valid when it's passed in.